src/patterns-hmrc-govuk-team/notices/index.njk

```diff
@@ -41,23 +37,27 @@ Do not include:
 <li>'Other notices on this or related subjects'</li>
 </ul>
 
+<h3 class="govuk-heading-m">Document type</h3>
+
+<p class="govuk-body">Detailed guide</p>
+
 <h3 class="govuk-heading-m">Soft Drinks Industry Levy</h3>
 
-<h4 class="govuk-heading-s">Title format</h4>
+<h4 class="govuk-heading-s">Title</h4>
 
-<p class="govuk-body">Description (notice xx)</p>
+<p class="govuk-body">Enter '[Description] (notice [number])'.</p>
 
 <h3 class="govuk-heading-m">Excise and VAT only</h3>
 
-<h4 class="govuk-heading-s">Title format</h4>
+<h4 class="govuk-heading-s">Title </h4>
```

Comment:
Tiny thing but could you remove the space after the word 'Title'?

Reply:
Done :)